### ISSUE TITLE
Localstack failing on amazon_kclpy import

### ISSUE THREAD

**Issue by imincik** [localstack/localstack#1033]
When installed using pip, Localstack 0.8.8 is failing as below. This doesn't happen with Localstack 0.8.7

```
travis@travis-job-0a14abf2-16ac-48aa-8b19-fe048f75cae7:~/build/linz/bde-processor-deployment$ sudo -u travis SERVICES="s3" /opt/localstack/bin/localstack --debug start
Traceback (most recent call last):
  File "/opt/localstack/bin/localstack", line 30, in <module>
    from localstack.services import infra
  File "/opt/localstack/lib/python3.4/site-packages/localstack/services/infra.py", line 26, in <module>
    from localstack.services.firehose import firehose_api
  File "/opt/localstack/lib/python3.4/site-packages/localstack/services/firehose/firehose_api.py", line 17, in <module>
    from localstack.utils.kinesis import kinesis_connector
  File "/opt/localstack/lib/python3.4/site-packages/localstack/utils/kinesis/kinesis_connector.py", line 11, in <module>
    from amazon_kclpy import kcl
ImportError: No module named 'amazon_kclpy'
```

**Comment by ilyanoskov** [comment]
I am experiencing the same issue

**Comment by matt-vib** [comment]
Same here, with pip version 18.1 and localstack version 0.8.8

**Comment by taxpon** [comment]
Same here (pip: 18.0, localstack: 0.8.8) and after installing `amazon_kclpy` manually (`pip install amazon_kclpy`), it works.

**Comment by mgab** [comment]
Same here. Since #266, `amazon_kclpy` is not installed with localstack with pip. Apparently, [this was done](https://github.com/localstack/localstack/issues/263#issuecomment-325193345) to have the package installed only for develompent, but not for normal usage.

However, #875 added `amazon_kclpy` as an import in `localstack/services/firehose/firehose_api.py` for v0.8.8. Since then you need the package but it is not installed as dependency, which I'd say is a bug.

I did the research, but I don't understand the reasons why the package was removed from the installed packages or why the import was needed, so I don't know what's the best fix. Maybe the authors of the PR, @whummer and @derrick-mink-sp, can add some insight? 😃

**Comment by michaeljohnbennett** [comment]
I would have hoped this kind of bug would have been picked up in your CI pipeline prior to release guys... 😉

**Comment by derrick-mink-sp** [comment]
This feature allows firehose to subscribe to a Kinesis stream as it does in AWS actual, so importing kinesis_connector made sense. That being said, we were unaware the underlying amazon_kclpy pkg was only being used for development & testing

**Comment by whummer** [comment]
Thanks for reporting @imincik - this should be fixed in the latest version (`0.8.10`). Closing this issue - please re-open if the problem persists. Thanks

**Comment by imincik** [comment]
Thanks @whummer
